lib/ansible/plugins/doc_fragments/constructed.py

```diff
@@ -79,4 +79,18 @@ class ModuleDocFragment(object):
     type: boolean
     default: True
     version_added: '2.11'
+  use_vars_plugins:
```

Comment:
don't move use_vars_plugins, this is not in use by all the plugins that subclass the generic constructed base class.

Reply:
Sorry that was a misunderstanding. I moved it back to the plugin itself.